docs/conf/atlantic/2024/index.rst

```diff
@@ -0,0 +1,29 @@
+:template: {{year}}/index.html
+:banner: _static/conf/images/headers/{{ shortcode }}-group.png
+
+:orphan:
+
+.. title:: Home | Write the docs {{ name }} {{ year }}
+
+
+.. raw:: html
+
+  <div class="news-block">
+    <div class="uk-container">
+
+      <h2>Updates from the team</h2>
+      <section>
+      <div class="content">
+
+.. postlist:: 10
```

Comment:
There's a strange block of text titled "Past attendees have said", which is aligned and formatted a bit off. I'm not sure where the snippet comes from but I don't remember including this information in the landing page in the past and I'm not sure if it fits here...